com.snipaste.Snipaste.metainfo.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.snipaste.Snipaste</id>
+  <name>Snipaste</name>
+  <project_license>LicenseRef-proprietary=https://www.snipaste.com/index.html</project_license>
+  <developer_name>Snipaste</developer_name>
+  <summary>Snipaste is not just a snipping tool. It will boost your productivity if you get used to its 'paste'!</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <launchable type="desktop-id">com.snipaste.Snipaste.desktop</launchable>
+  <url type="homepage">https://www.snipaste.com</url>
+  <url type="help">https://www.snipaste.com/support.html</url>
+  <description>
+    <p>Snipaste is not just a snipping tool. It will boost your productivity if you get used to its 'paste'!</p>
+    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Snipaste.</p>
```

Comment:
```suggestion
    <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Snipaste.</p>
    <p>Snipaste is not just a snipping tool. It will boost your productivity if you get used to its 'paste'!</p>
```

Make this the first p tag.

Reply:
fixed